setup.cfg

```diff
@@ -22,7 +22,7 @@ packages = find:
 setup_requires =
   setuptools-scm
 install_requires =
-  flask==1.1.4
+  flask==2.2.2
```

Comment:
Hmm, this thing is a library, so it should be as relaxed as possible with dependencies. (Our app deployments should be the only ones pinning.)

Reply:
I updated the range to `>=1`, if I should be more restrictive let me know